osu.Game.Rulesets.Taiko/Mods/TaikoModPlayfieldResize.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Layout;
+using osu.Game.Beatmaps;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Taiko.Objects;
+using osu.Game.Rulesets.Taiko.UI;
+using osu.Game.Rulesets.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModPlayfieldResize : Mod, IApplicableToDrawableRuleset<TaikoHitObject>, IApplicableToDifficulty
+    {
+        public override string Name => "Playfield Resize";
+        public override string Acronym => "PR";
+        public override string Description => @"Resize the playfield to see more or less notes";
+        // Ideally, this should go to System and be configurable from settings or something
+        public override ModType Type => ModType.Fun;
+        public override double ScoreMultiplier => 1;
+
+        public override string SettingDescription => PlayfieldScale.IsDefault ? "4:3" : $"Playfield Scale {PlayfieldScale.Value}";
+
+        [SettingSource("Playfield Scale", "The new playfield size")]
+        public BindableNumber<double> PlayfieldScale { get; } = new BindableDouble
+        {
+            MinValue = 0.5,
+            MaxValue = 2,
+            // 0.7 is equivalent to osu!stable's 4:3 play area
+            Default = 0.7,
```

Comment:
How was this derived?

Reply:
Manually. osu!lazer's playfield has more notes to the left of the hit area, and that seems to make the equivalent to 4:3 0.7 rather than 0.75. Maybe I'm wrong about the reasoning, but either way, 0.75 showed too many notes, and 0.7 *seems* just right.

Reply:
Manually. osu!lazer's playfield has more notes to the left of the hit area, and that seems to make the equivalent to 4:3 0.7 rather than 0.75. Maybe I'm wrong about the reasoning, but either way, 0.75 showed too many notes, and 0.7 *seems* just right.

Reply:
i actually calculated this properly now!